src/routes/request.js

```diff
@@ -11,13 +12,16 @@ const {
   oneWay,
   multiCityRequest,
   getRequests,
+  availOpenRequests,
   returnRequest
 } = RequestController;
 
 router.post('/multi-city', [auth, validator(requestSchema)], multiCityRequest);
 router.post('/one-way', [auth, validator(requestSchema)], oneWay);
 router.patch('/:requestId', [auth, validator(requestSchema)], update);
+router.get('/pending', auth, permitUser(['manager']), availOpenRequests);
```

Comment:
a `super admin` should be able to access this route `permitUser['manager', 'super_admin']`

Reply:
@amoskeyz this is a manager's role. I don't think this suits a super admin unless he would be assigned as manager to a department as well. I think a super admin should have another route where he can get all requests not just for direct reports. Do you see my point ,?

Reply:
not really. According to the PO she said a super admin should be able to access all route in the app.

Reply:
ok... i get the point now